Cargo.toml

```diff
@@ -39,6 +48,9 @@ defmt-03 = ["dep:defmt"]
 # Enable larger MPMC sizes.
 mpmc_large = []
 
+# Implement AlignedVec
+aligned_vec = []
```

Comment:
there's no need to gate it behind a feature, the only reason to do it is compile time and heapless already compiles really fast.